upi/openstack/network.yaml

```diff
@@ -28,6 +28,18 @@
       allocation_pool_end: "{{ os_subnet_range | ansible.utils.ipaddr('last_usable') }}"
       dns_nameservers: "{{ os_external_dns }}"
 
+  - name: 'Create the primary cluster IPv6 subnet'
+    openstack.cloud.subnet:
+      name: "{{ os_subnet6 }}"
+      network_name: "{{ os_network }}"
+      cidr: "{{ os_subnet6_range }}"
```

Comment:
I think there is a typo on the IPv6 CIDR defined on the inventory, can you update it?
https://github.com/openshift/installer/blob/master/upi/openstack/inventory.yaml#L10